src/main/java/de/craften/ui/swingmaterial/MaterialMenuItem

```diff
@@ -0,0 +1,59 @@
+package de.craften.ui.swingmaterial;
+
+import de.craften.ui.swingmaterial.MaterialShadow;
+import de.craften.ui.swingmaterial.fonts.Roboto;
+import java.awt.Color;
+import java.awt.Dimension;
+import javax.swing.Action;
+import javax.swing.BorderFactory;
+import javax.swing.Icon;
+import javax.swing.JMenuItem;
+import javax.swing.UIManager;
+import javax.swing.plaf.basic.BasicMenuItemUI;
+
+/**
+ *
```

Comment:
Please add some javadoc here, as in the previous PR. Javadoc for the public constructors would also be a good thing.